requirements.txt

```
@@ -403,7 +403,7 @@ sphinx-notfound-page==0.8.3
     # via plasmapy (setup.py)
 sphinx-reredirects==0.1.2
     # via plasmapy (setup.py)
-sphinx-rtd-theme==1.2.0
+sphinx-rtd-theme==1.2.1
```

```suggestion
sphinx-rtd-theme==1.2.0
```
...to trigger checks.